uswds/uswds
# Rewrite navigate_far icons to allow customColor

This PR rewrites the `navigate_far_before` and `navigate_far_next` icons to remove their custom fill and match other icons in the set that allow a customColor fill.

## How to check

1. Open the [icon component page in Storybook](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/dw-icons-currentColor/iframe.html?args=&id=design-tokens-icons--icons&viewMode=story)
2. With the web inspector, find the `fill` property on `.usa-icon` and set a custom color like `red`
3. Check that `navigate_far_before` and `navigate_far_next` are red


Fixes #4752
